src/backend/launcher.ts

```diff
@@ -143,11 +143,7 @@ async function prepareLaunch(
     }
   }
 
-  if (
-    (gameSettings.gamescope?.enableLimiter ||
-      gameSettings.gamescope?.enableUpscaling) &&
-    !isSteamDeckGameMode
-  ) {
+  if (gameSettings.gamescope?.enableGamescope && !isSteamDeckGameMode) {
```

Comment:
with this change we'll be disabling gamescope for all users that currently have it configured wit the other checkboxes until they go and manually enable it

Maybe we need some code setting the initial `enableGamescope` setting to be the value of `gameSettings.gamescope?.enableLimiter || gameSettings.gamescope?.enableUpscaling`. Or maybe we can type it as `boolean | null` with `null` meaning `it was never set, use the previous logic`?